x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -21,7 +21,9 @@ export const APP_ID = 'securitySolution' as const;
 export const APP_UI_ID = 'securitySolutionUI' as const;
 export const ASSISTANT_FEATURE_ID = 'securitySolutionAssistant' as const;
 export const ATTACK_DISCOVERY_FEATURE_ID = 'securitySolutionAttackDiscovery' as const;
-export const CASES_FEATURE_ID = 'securitySolutionCases' as const;
+// TODO: Remove old CASES_FEATURE_ID reference
+// export const CASES_FEATURE_ID = 'securitySolutionCases' as const;
```

Comment:
Not sure if there's a reason to keep this older ref around in security?

As an aside, it _feels_ like stack, obs, and security _should_ stay in sync with any permission changes..It might be better to just co-locate all these feature ids...

Reply:
Agreed, is there a reason this is kept as commented out?